common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -133,6 +133,7 @@ public ScoreResponse score(ScoreRequest request) {
     } catch (Exception e) {
       log.info("Failed shapley contribution due to: {}", e.getMessage());
       log.debug(" - failure cause: ", e);
+      throw new ShapleyScoreException(e.getMessage());
```

Comment:
It's (generally) good practice to keep the stack of original exceptions when throwing.  Most exception classes do this by accepting another exception as part of a custom exception's constructor.

Or, if you end up modifying/removing/etc. the custom exception type, you can just wrap the original exception with whichever `RuntimeException` [subtype](https://programming.guide/java/list-of-java-exceptions.html) makes sense.

Reply:
removed `ShapleyScoreException` as unneeded, so this is more or less resolved.